client-runtime/src/main/java/com/microsoft/rest/v2/annotations/Headers.java

```diff
@@ -0,0 +1,22 @@
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for
+ * license information.
+ */
+
+package com.microsoft.rest.v2.annotations;
+
+import java.lang.annotation.Retention;
+import java.lang.annotation.Target;
+
+import static java.lang.annotation.ElementType.METHOD;
+import static java.lang.annotation.RetentionPolicy.RUNTIME;
+
+@Retention(RUNTIME)
+@Target(METHOD)
+public @interface Headers {
+    /**
+     * List of static headers.
+     */
+    String[] value();
```

Comment:
Do these strings take the format "headerName=headerValue1,headerValue2"? Does that mean that the annotation parser will have to parse the list of headers?